src/Symfony/Component/DependencyInjection/Tests/Compiler/ValidateEnvPlaceholdersPassTest.php

```diff
@@ -340,6 +341,7 @@ class ConfigurationWithArrayNodeRequiringOneElement implements ConfigurationInte
 {
     public function getConfigTreeBuilder()
     {
+        throw new \Exception();
```

Comment:
the test ensures we dont require a node to be present, if the user didnt configure anything. However the side effect is the config tree isnt even initialized ... so it doesnt matter much what happens here.

I'd keep this tree for the purpose of clarity, in the sense we don't expect an `InvalidConfigException` due `nodes` being required.